src/pages/LoginPage/LoginForm.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export default function LoginForm({ userLoginInputs, onChange, onSubmit }) {
+  const { email, password } = userLoginInputs;
```

Comment:
```suggestion
export default function LoginForm({ email, password, onChange, onSubmit }) {
```
userLoginInputs 자체로는 사용되지 않고, 구조분해할당하여 사용하는 것 같아요. 

애초에 각각 넘겨주는 방식도 괜찮을 것 같네요~

Reply:
로그인폼에 인자가 많아지는 것 같아서 관련된 것은 useLoginInputs으로 두었습니다. 
state에서 둘은 같이 있어서 가독성도 좋아보이는데 분해해서 해야할까요..?

Reply:
말씀하신 이유라면 지금도 괜찮을 것 같습니다~ 😊

개인적으로 `userLoginInputs`이라는 객체명이 명확하지 않아 보이기도하고 이런 상황에는 구조분해 할당을 해서 넘기는걸 선호해서 커맨트 드렸습니다 ㅎㅎ

Reply:
아 제가 잘못 봤었어요.. 저는 initialState를 각각 따로 분리하라고 말씀하신줄 알았습니다. 수정할게요!😊